advocacy_docs/supported-open-source/pgbackrest/08-multiple-repositories.mdx

```diff
@@ -0,0 +1,591 @@
+stanza: demo
+    status: ok
+    cipher: none
+
+    db (current)
+        wal archive min/max (13): 000000010000000000000016/000000010000000000000019
+
+        full backup: 20210419-143643F
+            timestamp start/stop: 2021-04-19 14:36:43 / 2021-04-19 14:37:01
+            wal start/stop: 000000010000000000000016 / 000000010000000000000016
+            database size: 63.0MB, database backup size: 63.0MB
+            repo1: backup set size: 10MB, backup size: 10MB
+
+        full backup: 20210419-143749F
+            timestamp start/stop: 2021-04-19 14:37:49 / 2021-04-19 14:38:03
+            wal start/stop: 000000010000000000000018 / 000000010000000000000018
+            database size: 63MB, database backup size: 63MB
+            repo2: backup set size: 10MB, backup size: 10MB
+```
+
+As you can see in the example above, the `backup` command automatically triggers the `expire` command on the repository where we took our backup. That means it would not expire anything in the other repositories unless you request a new backup there or execute the `expire` command manually.
+
+Since the default retention policy is based on a number of backups, it is safe and healthy that only new backups are affecting the backups and archives to expire.
+
+#### Restore Command
+
+The restore command automatically defaults to selecting the latest backup from the first repository where backups exist. The order in which the repositories are checked is dictated by the `pgbackrest.conf` (e.g. repo1 will be checked before repo2). To select from a specific repository, the `--repo` option can be used.
+
+For PITR, `--type=time` must be provided and the target time must be specified with the `--target` option. If a backup is not specified via the `--set` option, then the configured repositories will be checked for a backup that contains the requested time. If no backup can be found, the latest backup from the first repository containing backups will be used.
+
+```bash
+$ mkdir /tmp/restored_data
+$ pgbackrest restore --stanza=demo --target="RP1" --type=name --no-delta --pg1-path=/tmp/restored_data
+P00   INFO: repo1: restore backup set 20210419-143643F
+
+$ rm -rf /tmp/restored_data/*
+$ pgbackrest restore --stanza=demo --target="2021-04-19 14:38:55.313716+00:00" --type=time --no-delta --pg1-path=/tmp/restored_data
+HINT: time format must be YYYY-MM-DD HH:MM:SS with optional msec and optional timezone (+/- HH or HHMM or HH:MM) 
+	- if timezone is omitted, local time is assumed (for UTC use +00)
+P00   INFO: repo1: restore backup set 20210419-143643F
+
+$ rm -rf /tmp/restored_data/* 
+$ pgbackrest restore --stanza=demo --target="RP1" --type=name --no-delta --pg1-path=/tmp/restored_data --repo=2
+P00   INFO: repo2: restore backup set 20210419-143749F
+```
+
+Even if the backup in **repo2** is newer, the first match found for our time target is kept. Let's take a new backup in **repo1** to check if pgBackRest will auto-select the backup in **repo2**:
```

Comment:
Even if the backup in **repo2** is newer, the first found match is preserved.

Reply:
Fixed in 7b5477c